loader/build.gradle

```diff
@@ -27,7 +27,7 @@ dependencies {
     implementation("net.neoforged.accesstransformers:at-modlauncher:${accesstransformers_version}") {
         exclude module: 'modlauncher'
     }
-    implementation("org.spongepowered:mixin:${mixin_version}")
+    implementation("net.fabricmc:sponge-mixin:${mixin_version}")
```

Comment:
I assume this is available on maven central?

Reply:
It is not, I have set up a proxy for that specific group on our maven.

Reply:
Even better!